### PR TITLE
DBZ-1184 Support long purged GTID set

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java [hunk 1]
```diff
@@ -205,7 +205,7 @@ public String knownGtidSet() {
     public GtidSet purgedGtidSet() {
         AtomicReference<String> gtidSetStr = new AtomicReference<String>();
         try {
-            jdbc.query("SHOW GLOBAL VARIABLES LIKE 'gtid_purged'", rs -> {
+            jdbc.query("SELECT @@global.gtid_purged", rs -> {
                 if (rs.next() && rs.getMetaData().getColumnCount() > 1) {
```
Comment (seruman) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java]: When I tried on MySQL 5.7, `SELECT @@global.gtid_purged` returned only the value column unlike `SHOW` which returns key and value columns.

Comment (jpechane) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java]: @seruman Thanks for pointing out, I've added a test to assert the codepath

**Comment by gunnarmorling** [comment]
LGTM; will merge once CI has passed.

**Comment by gunnarmorling** [comment]
There's still a test failure, @jpechane.

**Comment by jpechane** [comment]
@gunnarmorling I know, the problem is that flushing binlog can cause failing of unelated tests and this unfortunately depends on test class order execution we have no way to influence. So I am now investigating two options
 - somehow force specifing ordernign od the tests
 - introducing a specific test group that would run after the other tests were executed

**Comment by jpechane** [comment]
This looks like a solution - http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#runOrder

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/ZZZGtidSetIT.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import io.debezium.jdbc.JdbcConnection;
+import io.debezium.util.Testing;
+
+/**
+ * The test is named to make sure it runs alphabetically last as it can influence execution
+ * of for example {@link BinlogReaderIT}
+ *
+ * @author Jiri Pechanec
+ */
+public class ZZZGtidSetIT extends AbstractConnectorTest {
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/ZZZGtidSetIT.java]: Uff, ok, that's kinda... ugly :)

Comment (jpechane) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/ZZZGtidSetIT.java]: No that I am superrhappy about that, but it works and I inteded to visually make it declared it is in the end. It was working when the code was in MySsqlConnectorIT as the alphabetical order also gurateed it but I did not like to hide dependencies/.

**Pull request merged**